zebrad/src/application.rs

```diff
@@ -155,17 +155,19 @@ impl Application for ZebradApp {
         };
 
         // collect the common metadata for the issue URL and panic report
-        let network = config.network.network.to_string();
         let panic_metadata = vec![
-            ("version", env!("CARGO_PKG_VERSION").to_string()),
-            ("git commit", Self::git_commit().to_string()),
-            ("Zcash network", network),
+            ("version", env!("CARGO_PKG_VERSION")),
+            ("git commit", Self::git_commit()),
+            ("commit timestamp", env!("VERGEN_GIT_COMMIT_TIMESTAMP")),
+            ("target triple", env!("VERGEN_CARGO_TARGET_TRIPLE")),
+            ("branch", env!("VERGEN_GIT_BRANCH")),
+            ("Zcash network", (&config.network.network).into()),
```

Comment:
I just realised that the order of these variables is preserved in the panic `metadata_section`. So let's keep them roughly sorted, to make bug reports a bit easier to read:
```suggestion
            // git
            ("version", env!("CARGO_PKG_VERSION")),
            ("branch", env!("VERGEN_GIT_BRANCH")),
            ("git commit", Self::git_commit()),
            ("commit timestamp", env!("VERGEN_GIT_COMMIT_TIMESTAMP")),
            // build
            ("target triple", env!("VERGEN_CARGO_TARGET_TRIPLE")),
            // config
            ("Zcash network", (&config.network.network).into()),
```

Reply:
Thanks 👍